README.md

```diff
@@ -23,17 +23,33 @@ There are two components to keep in mind when trying to use this application.
 
 ## Software Requirements
 1. [poetry](https://python-poetry.org/)
-2. Python 3.9
+    - https://python-poetry.org/docs/#installing-with-the-official-installer 
```

Comment:
@sujaypatil96 there might be a better way to show both links? Or maybe one is better than the other. But I found the 2nd poetry link to be more helpful

Reply:
Yes, the second link is better in that it's a direct page to the "installation instructions". We should replace the first link (https://python-poetry.org/) with the new one (https://python-poetry.org/docs/#installing-with-the-official-installer)